Sensor_Temperature.dcm

```diff
@@ -462,6 +462,18 @@ K I2C Temperature Sensor
 F https://www.silabs.com/documents/public/data-sheets/Si7050-1-3-4-5-A20.pdf
 $ENDCMP
 #
+$CMP TC1047xNB
+D Precision Temperature-to-Voltage Converter, 10mV/ºC, -40ºC to +125ºC, ±2.0ºC (max), 2.7V to 4.4V, SOT-23B
```

Comment:
Change package to just SOT-23

Reply:
although SOT-23B is mentioned in the datasheet?

Reply:
Yes.
SOT-23B seems to be Microchip's way of referring to TO-236 variation AB (which differs from AA in height tolerance), but that's neither salient nor does the lib contain different FPs for the two variants.